hack/tools/go.mod

```diff
@@ -0,0 +1,9 @@
+module sigs.k8s.io/kind/hack/tools
+
+go 1.13
+
+require (
+	github.com/bazelbuild/bazel-gazelle v0.19.1-0.20191105222053-70208cbdc798
+	github.com/bazelbuild/buildtools v0.0.0-20190917191645-69366ca98f89
+	k8s.io/repo-infra v0.0.1-alpha.1
+)
```

Comment:
was this tidied with `go mod tidy`? Does anything make sure that is done before PRs merge?

Reply:
the verify script runs hack/update-bazel ... which fetches the dependencies ... which ends up updating go.sum. So the verify job may catch a dirty go.sum? (need to confirm)

Reply:
That wouldn't tidy though?

Reply:
i'd vote for adding update/verify later. honestly these don't change quickly and one of us can help for now.

Reply:
(and tidy)

Reply:
added update/verify scripts.